internal/bus/serialization.go

```diff
@@ -49,7 +49,10 @@ func deserializeChannel(b []byte) (string, error) {
 
 func deserialize(b []byte) (proto.Message, error) {
 	a := &anypb.Any{}
-	err := proto.Unmarshal(b, a)
+	opt := proto.UnmarshalOptions{
+		DiscardUnknown: true,
```

Comment:
interesting, I had thought this would be true by default. we use unmarshal in quite a bit of places and most assume future-proof Unmarshaling

Reply:
protobufs are forward/backward compatible but the undecoded fields get stored by default - i think they can be accessed via reflection. but we don't need them here.

the new envelope message is wire compatible with `anypb.Any` and when i decode the `channel` from it i don't want to pay for decoding the rest of it into whatever internal structure stashes the undefined fields... and we might as well do the same when we decode the `anypb.Any`